clients/src/test/java/org/apache/kafka/clients/consumer/internals/AbstractStickyAssignorTest.java

```diff
@@ -555,7 +558,7 @@ public void testLargeAssignmentAndGroupWithUniformSubscription() {
         assignor.assign(partitionsPerTopic, subscriptions);
     }
 
-    @Timeout(40)
+    @Timeout(60)
```

Comment:
Saw this fail sometimes when running locally. This PR definitely should not have significantly affected the performance of this test in any way, since the only substantial changes are in the constrained assignment algorithm, while this test targets the general case of unequal subscriptions. It's not too surprising to me that this would be running up against the time limit anyways, since we've known that the general assignment algorithm is complicated and doesn't scale too well.So I felt it was reasonable to give it a bit more time. 

Alternatively we could just scale back a bit with the number of partitions and/or consumers. WDYT @showuon?

Reply:
Increase the timeout makes sense to me.

Reply:
+1